elements/bulbs-video/elements/summary.js

```diff
@@ -24,6 +24,19 @@ export function VideoSummaryView (props) {
     );
   }
 
+  let campaign;
+  if (video.tunic_campaign_url) {
+    campaign = (
+      <campaign-display
+          data-track-action={props.campaignTrackAction}   // eslint-disable-line react/prop-types
```

Comment:
why the eslint-disable? does it want you to do `const { campaignTrackAction, campaignPlacement, campaignPreamble } = props`? (cleaner imo)